src/Compilers/Core/CodeAnalysisTest/ObjectSerializationTests.cs

```diff
@@ -17,14 +17,6 @@ namespace Microsoft.CodeAnalysis.UnitTests
 {
     public sealed class ObjectSerializationTests
     {
-        static ObjectSerializationTests()
-        {
-            // Register appropriate deserialization methods.
-            new PrimitiveArrayMemberTest();
-            new PrimitiveMemberTest();
-            new PrimitiveValueTest();
-        }
```

Comment:
i deleted all the tests related to object graph serialization.  all the tests testing things like writing out primitives/arrays/strings/etc. all remain and are still relevant.